code/game/gamemodes/miniantags/borer/borer.dm

```diff
@@ -81,7 +79,11 @@
 	pass_flags = PASSTABLE
 	ventcrawler = 2
 	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
```

Comment:
Since this is a TG port, make sure all the borers vars/stats match up with them.